src/csharp/Microsoft.Spark/Sql/Types/DataType.cs

```diff
@@ -56,10 +55,10 @@ public string Json
         {
             get
             {
-                object jObject = (JsonValue is JObject) ?
-                    ((JObject)JsonValue).SortProperties() :
+                object jObject = (JsonValue is JsonElement) ?
```

Comment:
This can be re-written so you only have a single cast:

```
object jObject = (JsonValue is JsonElement jsonElement) ?
    jsonElement.SortProperties() :
    JsonValue;
```